independent-projects/resteasy-reactive/server/vertx/src/main/java/org/jboss/resteasy/reactive/server/vertx/ResteasyReactiveOutputStream.java

```diff
@@ -80,7 +82,7 @@ Buffer createBuffer(ByteBuf data) {
 
     public void write(ByteBuf data, boolean last) throws IOException {
         if (last && data == null) {
-            request.response().end();
+            request.response().end(EMPTY_BUFFER, null);
```

Comment:
```suggestion
            request.response().end((Handler<AsyncResult<Void>>) null);
```

You could save the empty buffer creation like this. Vert.x does it internally.